tabs/applications-setup/bottles-setup.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh -e
+
+. ../common-script.sh
+
+install_bottles() {
+    printf "%b\n" "${YELLOW}Installing Bottles...${RC}"
+    if ! command_exists flatpak; then
+    case $PACKAGER in
+        pacman)
+            $ESCALATION_TOOL ${PACKAGER} -Syu --noconfirm flatpak         
+            ;;
+        apt-get)
+            $ESCALATION_TOOL ${PACKAGER} update && $ESCALATION_TOOL ${PACKAGER} install -y flatpak
+            ;;
+        dnf)
+            $ESCALATION_TOOL ${PACKAGER} install -y flatpak
+            ;;
+        zypper)
+            $ESCALATION_TOOL ${PACKAGER} install flatpak
+            ;;
+        *)
+            printf "%b\n" "${RED}Your Linux distribution is not supported by this script.${RC}"
+            printf "%b\n" "${YELLOW}You can try installing Bottles using Flatpak manually:${RC}"
+            printf "%b\n" "1. Install Flatpak: https://flatpak.org/setup/"
+            printf "%b\n" "2. Install Bottles: flatpak install flathub com.usebottles.bottles"
+            ;;
+    esac
+    fi
+    flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
```

Comment:
Perhaps replace this with a call to the flatpak setup script that was added.
```suggestion
    . ./setup-flatpak.sh
```